src/routes/routes.ts

```diff
@@ -16,5 +16,6 @@ const renderTemplate = (template: string) => (req: Request, res: Response, next:
 
 router.get(pageURLs.ROOT, renderTemplate(templatePaths.INDEX));
 router.get(pageURLs.COMPANY_NUMBER, renderTemplate(templatePaths.COMPANY_NUMBER));
+router.get("/check-company", checkCompanyRoute.route);
```

Comment:
Should this URL be in pageURLs as a constant?

Reply:
Yes it should - it will be worked on immediately afterwards on ticket lfa-1235